### PR TITLE
Add componentDidCatch from React 16

### PR DESCRIPTION
**Pull request by mclark4386**
Should add componentDidCatch snippet as _cdc to help with taking advantage of this part of React 16.

### CODE REVIEW

File: snippets/React (JSX).cson [hunk 1]
```diff
@@ -31,6 +31,10 @@
     prefix: "_cdm"
     body: "componentDidMount() {\n\t${1}\n}${2}"
 
+  "React: componentDidCatch(error, info) { ... }":
+    prefix: "_cdc"
+    body: "componentDidUpdate(error, info) {\n\t${1}\n}${2}"
```
Comment (webbushka) [inline snippets/React (JSX).cson]: @mclark4386 looks like the body won't be what you're expecting

**Review by webbushka — CHANGES_REQUESTED** [review]

File: snippets/React (JSX).cson (hunk 1 above)
Comment (mclark4386) [reply snippets/React (JSX).cson]: oh man... derp, sorry

**Comment by mclark4386** [comment]
Should be changed now, sorry about that!

**Comment by webbushka** [comment]
thanks!

**Pull request merged**